nuxeo-data-table/iron-data-table.html

```diff
@@ -472,15 +472,22 @@ <h2>[[i18n('command.add')]]</h2>
               type: Boolean,
               value: false,
             },
+            /**
+             * Extended Nuxeo-Data-Table-Checkox tag name
+             */
+            checkboxTagName: {
```

Comment:
We'd likely not expose this as a public property in any case, just `_tagName` would be better.

Reply:
done

Reply:
renamed to _checkboxTagName to reflect the purpose of this property and be "private"